config-diff

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ] ; then
```

Comment:
Should we include this in the README?
I can do it later.

Reply:
Oh word! Yeah actually I was planning on making more changes when I get the time, specifically changing the completion scripts (Be able to add to .bashrc/.zshrc, pull list of commons from Github API instead of using the hardcoded list). I can update the README with this + installation instructions once I do that.